app/models/drupal_node.rb

```diff
@@ -412,17 +440,17 @@ def edit_path
     path
   end
 
-  def self.find_by_slug(title)
-    DrupalNode.where(path: ["/#{title}", "/tool/#{title}", "/wiki/#{title}", "/place/#{title}"]).first
-  end
+  # def self.find_by_slug(title)
```

Comment:
Well, both because the map URLs are specifically designed to have dates,
and because we need to support legacy URLs for existing maps, I think we
need to match the existing map URL format exactly. Thanks, Ananyo, I
appreciate the extra effort.

On Jul 22, 2016 1:23 PM, "Ananya Maiti" <notifications@github.com> wrote:

> In app/models/drupal_node.rb
> <https://github.com/publiclab/plots2/pull/600#discussion_r71914211>:
>
> > @@ -412,17 +440,17 @@ def edit_path
> >      path
> >    end
> >
> > -  def self.find_by_slug(title)
> > -    DrupalNode.where(path: ["/#{title}", "/tool/#{title}", "/wiki/#{title}", "/place/#{title}"]).first
> > -  end
> > +  # def self.find_by_slug(title)
>
> Yes I can delete it once you clarify the last point.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/plots2/pull/600/files/c968e35c5fa7b3232e5e5b2ca0bc9afa658586cd#r71914211>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ2ZwW0uVTbD0JOz6Sr1pVXtKjJW1ks5qYPyBgaJpZM4JDL2B>
> .
>